src/styles/ForecastSummary.css

```diff
@@ -0,0 +1,29 @@
+body {
+  font-size: 18px;
+  text-align: center;
+}
+
+.forecast-summary {
+  background-color: rgb(255, 226, 64);
+  width: auto;
+  height: auto;
+  border-radius: 10px;
+}
+
+.forecast-summary__date {
+  font-size: 20px;
+  font-weight: bold;
+}
+
+.forecast-summary__icon {
+  font-size: 35px;
+}
+
+button {
+  border: none;
+}
+
+button:hover {
```

Comment:
Generally speaking its better to target classnames rather than HTML elements themselves, in can lead to awkward bugs especially as an app gets bigger and bigger. Much better to add a classname to the button and target that instead